docs/how-to/custom-env.txt

```diff
@@ -171,17 +171,19 @@ image (using the ``FROM`` instruction).
 Here is an example of a ``Dockerfile`` that installs both ``conda``- and
```

Comment:
I'm not a big fan of the `# optional` comments because literally everything in the example is optional except maybe the FROM. What if we reworded the preamble here to be: "To simply add some additional dependencies so that they do not have to be installed at runtime, simply use a standard image as the base and add your dependencies. Here is an example of a ``Dockerfile`` that installs ``conda``-, ``pip``- and ``apt``-based dependencies.

Reply:
changed to separate the sections